src/main/java/com/codecool/terraformingmarsforum/service/UserService.java

```diff
@@ -12,4 +13,8 @@ public class UserService {
 
     @NonNull
     private UserDAO userDAO;
```

Comment:
I think this is still field injection, but is it your part anyways?

Reply:
Ask Bende about that, I dont want to change it, bc thats not my concept